punchbowl/level1/vignette.py

```diff
@@ -115,27 +115,29 @@ def generate_vignetting_calibration(path_vignetting: str, path_mask: str,
         bin_size = int(bin_size)
 
         values = np.array([float(v) for line in lines[1:] for v in line.split()])
-        vignetting_original = values[:num_bins**2].reshape((num_bins, num_bins))
+        vignetting = values[:num_bins**2].reshape((num_bins, num_bins))
 
-        vignetting_original[np.isnan(vignetting_original)] = 0
+        vignetting[vignetting > 1.2] = np.nan
 
-        vignetting_original[6,:] = 0
+        vignetting[13:15,:] = np.nan
+        vignetting[15:16,:] = np.min(vignetting[16:20,:], axis=0)
```

Comment:
Can we document these magic numbers and also make them a parameter of the function?